src/components/AddressSelect/AddressesSelect.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { KeyboardEventHandler, useEffect, useState } from 'react'
+import { InputActionMeta, OnChangeValue } from 'react-select'
+import CreatableSelect from 'react-select/creatable'
+
+import classNames from 'classnames'
+import isEthereumAddress from 'validator/lib/isEthereumAddress'
+
+import useFormatMessage from '../../hooks/useFormatMessage'
+import Username from '../User/Username'
+
+import './AddressesSelect.css'
+
+interface UserAddress {
+  readonly label: JSX.Element
+  readonly value: string
+}
+interface State {
+  readonly inputValue: string
+  readonly value: readonly UserAddress[]
+  errorKey?: string
+}
+
+type Validation = {
+  validate: (address: string) => boolean
+  errorKey: string
+}
+
+const components = {
+  DropdownIndicator: null,
+}
+
+const createUserAddress = (address: string): UserAddress => ({
+  label: <Username address={address} size="tiny" />,
+  value: address.toLowerCase(),
+})
+
+interface Props {
+  maxAddressesAmount: number
+  allowLoggedUserAccount?: boolean
+  setUsersAddresses: (addresses?: string[]) => void
+  isDisabled?: boolean
+}
+
+function AddressesSelect({ setUsersAddresses, isDisabled, maxAddressesAmount, allowLoggedUserAccount = false }: Props) {
```

Comment:
I think we can create a common component where logic from this component and CoAuthorSelect can be reused.

Reply:
Yes we can and should. 
It started as a dev tool and now it's like a relevant feature, but I'd rather do this in a different PR, WDYT?

Reply:
[#1183](https://github.com/decentraland/governance/issues/1183)